resize-observer-1/Overview.bs

```diff
@@ -385,8 +385,8 @@ Watching content rect means that:
 
 * observations will not be triggered by CSS transforms.
 
-Web content can also contain SVG elements. SVG Elements define <a>bounding box</a> instead of a content box.
-Content rect for <a>SVGGraphicsElement</a>s is a rect whose:
+Web content can also contain SVG elements. SVG elements that do not have associated CSS layout boxes define <a>bounding box</a> instead of a content box.
+Content rect for the <a>SVGGraphicsElement</a>s without CSS layout boxes is a rect whose:
```

Comment:
I don't think there's any `SVGGraphicsElement` that can have any CSS layout box?

Reply:
> I don't think there's any `SVGGraphicsElement` that can have any CSS layout box?

For example, the SVG element (which is a `SVGGraphicsElement` by definition) in the following code will have a CSS layout box:

```
<body><svg width="100" height="100"></svg></body>
```

Reply:
D'oh, I'm silly, I was aware of `<svg>` getting a CSS box, but I was looking at [this](https://searchfox.org/mozilla-central/search?q=public+SVGGraphicsElement&path=&case=false&regexp=false) and didn't realize that `SVGSVGElement` inherited from `SVGViewportElement`.

This looks perfect, thank you!